temporal-sdk/src/main/java/io/temporal/workflow/TimerOptions.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (C) 2022 Temporal Technologies, Inc. All Rights Reserved.
+ *
+ * Copyright (C) 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this material except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.temporal.workflow;
+
+import io.temporal.common.Experimental;
+import java.util.Objects;
+
+/** TimerOptions is used to specify options for a timer. */
+public final class TimerOptions {
+
+  public static TimerOptions.Builder newBuilder() {
+    return new TimerOptions.Builder();
+  }
+
+  public static TimerOptions.Builder newBuilder(TimerOptions options) {
+    return new TimerOptions.Builder(options);
+  }
+
+  public static TimerOptions getDefaultInstance() {
+    return DEFAULT_INSTANCE;
+  }
+
+  private static final TimerOptions DEFAULT_INSTANCE;
+
+  static {
+    DEFAULT_INSTANCE = TimerOptions.newBuilder().build();
+  }
+
+  public static final class Builder {
+    private String staticSummary;
+    private String staticDetails;
+
+    private Builder() {}
+
+    private Builder(TimerOptions options) {
+      if (options == null) {
+        return;
+      }
+      this.staticSummary = options.staticSummary;
+      this.staticDetails = options.staticDetails;
+    }
+
+    /**
+     * Single-line fixed summary for this timer that will appear in UI/CLI. This can be in
+     * single-line Temporal Markdown format.
+     *
+     * <p>Default is none/empty.
+     */
+    @Experimental
+    public Builder setStaticSummary(String staticSummary) {
```

Comment:
We are not calling it "static summary" on a timer, just summary. From a user POV it is unrelated to workflow static summary and details (even if it does use the same data type behind the scenes).